plugins/LilyPad/LilyPad.cpp

```diff
@@ -1217,23 +1227,14 @@ u8 CALLBACK PADpoll(u8 value)
         switch (value) {
             // CONFIG_MODE
             case 0x43:
-                if (pad->config) {
```

Comment:
I don't get why you moved the code here into an if/else in the following switch case. Couldn't it just be:
```
case 0x43:
    if (pad->config && padtype != neGconPad) {
        // blah blah
        return whatever;
    }
    // Fallthrough
case 0x42:
```